diff --git a/html.go b/html.go
index 2f0ad3b..b4d5980 100644
--- a/html.go
+++ b/html.go
@@ -948,3 +948,19 @@ func (options *Html) ensureUniqueHeaderID(id string) string {
 
 	return id
 }
+
+func (options *Html) Math(out *bytes.Buffer, equation []byte, inline bool) {
+	if inline {
+		out.WriteString("\\(")
+	} else {
+		out.WriteString("\\[")
+	}
+
+	attrEscape(out, equation)
+
+	if inline {
+		out.WriteString("\\)")
+	} else {
+		out.WriteString("\\]")
+	}
+}
diff --git a/inline.go b/inline.go
index 4483b8f..90f1e62 100644
--- a/inline.go
+++ b/inline.go
@@ -1152,3 +1152,49 @@ func helperTripleEmphasis(p *parser, out *bytes.Buffer, data []byte, offset int,
 	}
 	return 0
 }
+
+// LaTeX math surrounded by '$' or '$$'
+func math(p *parser, out *bytes.Buffer, data []byte, offset int) int {
+	data = data[offset:]
+
+	// inline math
+	if len(data) > 2 && data[1] != '$' {
+		// find the next '$'
+		end := 1
+		for end < len(data) && data[end] != '$' {
+			end++
+		}
+
+		// no matching delimiter?
+		if end == len(data) {
+			return 0
+		}
+
+		// render the inline math
+		if end != 0 {
+			p.r.Math(out, data[1:end], true)
+		}
+		return end + 1
+	}
+
+	// display math
+	if len(data) > 4 && data[1] == '$' && data[2] != '$' {
+		// find the next '$$'
+		end := 2
+		for end+1 < len(data) && (data[end] != '$' || data[end+1] != '$') {
+			end++
+		}
+
+		// no matching delimiter?
+		if end+1 == len(data) {
+			return 0
+		}
+
+		// render the display math
+		if end != 0 {
+			p.r.Math(out, data[2:end], false)
+		}
+		return end + 2
+	}
+	return 0
+}
diff --git a/latex.go b/latex.go
index 70705aa..5d789f5 100644
--- a/latex.go
+++ b/latex.go
@@ -330,3 +330,19 @@ func (options *Latex) DocumentHeader(out *bytes.Buffer) {
 func (options *Latex) DocumentFooter(out *bytes.Buffer) {
 	out.WriteString("\n\\end{document}\n")
 }
+
+func (options *Latex) Math(out *bytes.Buffer, equation []byte, inline bool) {
+	if inline {
+		out.WriteString("\\(")
+	} else {
+		out.WriteString("\\[")
+	}
+
+	out.Write(equation)
+
+	if inline {
+		out.WriteString("\\(")
+	} else {
+		out.WriteString("\\]")
+	}
+}
diff --git a/markdown.go b/markdown.go
index 1722a73..9fb17d2 100644
--- a/markdown.go
+++ b/markdown.go
@@ -43,6 +43,7 @@ const (
 	EXTENSION_AUTO_HEADER_IDS                        // Create the header ID from the text
 	EXTENSION_BACKSLASH_LINE_BREAK                   // translate trailing backslashes into line breaks
 	EXTENSION_DEFINITION_LISTS                       // render definition lists
+	EXTENSION_LATEX_MATH                             // latex inline and display math surrounded by '$' or '$$'
 	EXTENSION_JOIN_LINES                             // delete newline and join lines
 
 	commonHtmlFlags = 0 |
@@ -187,6 +188,7 @@ type Renderer interface {
 	TripleEmphasis(out *bytes.Buffer, text []byte)
 	StrikeThrough(out *bytes.Buffer, text []byte)
 	FootnoteRef(out *bytes.Buffer, ref []byte, id int)
+	Math(out *bytes.Buffer, equation []byte, inline bool)
 
 	// Low-level callbacks
 	Entity(out *bytes.Buffer, entity []byte)
@@ -385,6 +387,10 @@ func MarkdownOptions(input []byte, renderer Renderer, opts Options) []byte {
 		p.notesRecord = make(map[string]struct{})
 	}
 
+	if extensions&EXTENSION_LATEX_MATH != 0 {
+		p.inlineCallback['$'] = math
+	}
+
 	first := firstPass(p, input)
 	second := secondPass(p, first)
 	return second
