diff --git a/helpers/content.go b/helpers/content.go
index 3c81fcd3..fdd78d1d 100644
--- a/helpers/content.go
+++ b/helpers/content.go
@@ -131,6 +131,7 @@ var blackfridayExtensionMap = map[string]int{
 	"autoHeaderIds":          blackfriday.EXTENSION_AUTO_HEADER_IDS,
 	"backslashLineBreak":     blackfriday.EXTENSION_BACKSLASH_LINE_BREAK,
 	"definitionLists":        blackfriday.EXTENSION_DEFINITION_LISTS,
+	"latexMath":              blackfriday.EXTENSION_LATEX_MATH,
 	"joinLines":              blackfriday.EXTENSION_JOIN_LINES,
 }
 
@@ -264,7 +265,8 @@ func getMarkdownExtensions(ctx *RenderingContext) int {
 		blackfriday.EXTENSION_SPACE_HEADERS |
 		blackfriday.EXTENSION_HEADER_IDS |
 		blackfriday.EXTENSION_BACKSLASH_LINE_BREAK |
-		blackfriday.EXTENSION_DEFINITION_LISTS
+		blackfriday.EXTENSION_DEFINITION_LISTS |
+	    blackfriday.EXTENSION_LATEX_MATH
 
 	// Extra Blackfriday extensions that Hugo enables by default
 	flags := commonExtensions |
